src/plugins/vis_type_timeseries/server/lib/vis_data/annotations/build_request_body.ts

```diff
@@ -22,7 +23,7 @@ import { processors } from '../request_processors/annotations';
  * ]
  * @returns {Object} doc - processed body
  */
-export async function buildAnnotationRequest(...args) {
+export async function buildAnnotationRequest(...args: any[]) {
```

Comment:
Can we type it? Or at least replace it with `unknown`?

Reply:
updated